premium/usage.go

```diff
@@ -143,8 +143,10 @@ type BatchUpdater struct {
 	maxTimeBetweenFlushes time.Duration
 
 	// State
+	rows           uint32
+	tables         map[string]uint32
+	mutex          sync.Mutex
```

Comment:
```suggestion
	sync.Mutex
```
will allow locking on `BatchUpdater` (`b.Lock()` essentially)